clang/test/SemaCXX/cxx20-ctad-type-alias.cpp

```diff
@@ -108,8 +108,11 @@ struct Foo {
   Foo(T const (&)[N]);
 };
 
+// FIXME: Prefer non-canonical template arguments in the deduction guide?
```

Comment:
We have a open bug for this, https://github.com/llvm/llvm-project/issues/79798.

Reply:
Good to know that, remove the FIXME now.